setup.py

```diff
@@ -58,7 +58,7 @@
             "flake8-rst-docstrings",
             "isort",
             "pre-commit",
-            "pylint",
+            "pylint>=0.0.0",

```

Comment:
This is new, if I understand this correctly just having `pylint` was always strictly asking for the latest version and was causing dependency conflicts, and the change you have will allow to install any pylint version that satisfies the other dependencies constraints?

Reply:
This will be pinned 3.1.0. I temporarily changed this for looking into a GH runner issue